keyboards/id67/keymaps/thewerther/readme.md

```diff
@@ -0,0 +1,31 @@
+# IDOBAO ID67
```

Comment:
This file can be deleted.

Reply:
Would I put this file in userspace if I wanted to include it in the repo so I can link people to it if they want separate underglow for certain effects?

Reply:
It's a copy of the keyboard-level readme. You don't need it at all.